Lib/test/test_sysconfig.py

```diff
@@ -611,7 +611,7 @@ def test_always_set_py_gil_disabled(self):
         self.assertEqual(Py_GIL_DISABLED, support.Py_GIL_DISABLED)
 
     def test_abiflags(self):
-        # XXX: If this test fails on some platforms, maintainers should add/update
+        # If this test fails on some platforms, maintainers should add/update
         # the definition of the ABIFLAGS variable and make this test pass.
```

Comment:
```suggestion
        # If this test fails on some platforms, maintainers should update the
        # test to make it pass, rather than changing the definition of ABIFLAGS
```

Reply:
See my comment above.